SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -1124,6 +1124,7 @@ void USpatialSender::RetireEntity(const Worker_EntityId EntityId)
 		}
 		else
 		{
+			UE_LOG(LogSpatialSender, Log, TEXT("Sending retire entity request for %s with EntityId %lld, HasAuthority: %d"), *Actor->GetName(), EntityId, Actor->HasAuthority());
```

Comment:
Should we be explicit here and say we're sending a delete entity request instead of retire?

Reply:
I think that sounds good.  Maybe we could also output a Log for the other branch when we're adding a tombstone?

Reply:
Doesn't this logic get caught in the existing log?
ie. `LogSpatialActorChannel: Delete entity request on 9826. Has authority: 1`

Reply:
@m-samiec I think that was accidentally missed when this was added.  That being said, that log entry doesn't cover all cases of calling RetireEntity, and doesn't distinguish between tombstoning and actual deletion, so maybe there's still some room for improvement here.

Reply:
I think if we're going to include this, we need to remove or make verbose, the log in `USpatialActorChannel::DeleteEntityIfAuthoritative`

Reply:
agreed - I think that can be removed if we add this here, or vice versa.  Either approach is good with us.

Reply:
Yeah this log looks better - lets remove the other. cc: @KeyMaster- :)

Reply:
I've found the ifAuthoritative log not very clear anyway. Will remove that and update this one :)